code/controllers/subsystem/blackbox.dm

```diff
@@ -16,7 +16,8 @@ SUBSYSTEM_DEF(blackbox)
 	/// List of highest tech levels attained that isn't lost lost by destruction of RD computers
 	var/list/research_levels = list()
 	/// Associative list of any feedback variables that have had their format changed since creation and their current version, remember to update this
-	var/list/versions = list()
+	var/list/versions = list("traitor_objective" = 2,
+							"changeling_objective" = 2)
```

Comment:
On second thoughts, dont V2 these.

V2 is for changes in data formatting, this is just recording less stuff.

Reply:
Removed